engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -4128,6 +4129,7 @@ public void mainHelper() throws Exception
 
   public static void main(final String[] args) throws Exception
   {
+    LoggerUtil.addAppender("apexclient");
```

Comment:
The property name (regular or long) of the  attribute instance LOGGER_APPENDER should be passed via Java command line parameters, because the reference to the dynamic appender must exist when the apex client is starting.

I don't think the regular (attr.LOGGER_APPENDER) or long (apex.logger.appender) can conflict with the same logger appender parameter value in applications, because this Java parameter is not explicitly inherited from the client to application(s). I guess it will be good and consistent behavior to use the same logger appender parameter name in the apex client and applications.